src/event.js

```diff
@@ -894,7 +894,7 @@ if ( !support.submit ) {
 				// Node name check avoids a VML-related crash in IE (#9807)
 				var elem = e.target,
 					form = jQuery.nodeName( elem, "input" ) || jQuery.nodeName( elem, "button" ) ?
-						elem.form :
+						jQuery.prop( elem, "form" ) :
```

Comment:
To prevent premature reversion, I'd like to see a comment explaining why this line uses `jQuery.prop` instead of `elem.form` (e.g., like [this](https://github.com/jquery/jquery/commit/ae215fdcf843862dbddbdfd68117ac2926e35a42#diff-89c55bb1a12d31cb4eb5d007633dc33bR14)). It should probably reference gh-2332, and be preceded by `Support: IE <=8`.